frontend/src/components/home/carousel/carousel.css

```diff
@@ -58,3 +61,25 @@ span {
 .owl-nav button:focus {
   outline: none;
 }
+@media(max-width: 500px){
```

Comment:
leave one blank line between all two css classes

Reply:
On it!

Reply:
![image](https://user-images.githubusercontent.com/69422250/103338899-e7908700-4aa5-11eb-919d-2f61059fa3fd.png)